File: Assignment_No-27/Question_05.c
/*
135)write a program which accept string from user and display in reverse order.

input:MarvellouS

output:SuollevraM
*/

//////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Reverse(char *str)
{
  char *p=str;
  while(*p!='\0')
  {
    	p++;
  }
  p--;
  while(p>=str)
  {
  	printf("%c\t",*p);
  	p--;
  }
}

int main()
{


	char Arr[20];
	int iRet=0;

	printf("Enter a string:\n");
	scanf("%[^'\n']s",Arr);

	Reverse(Arr);
  
	return 0;
}

File: Assignment_No-37/Question_05.java
/*5)write a java program which accept N numbers from user and return product of all odd elements.
input:N:6
elements:15 66 3 70 10 88

output:45
*/

//////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*; 
class Number
{
   public int Arr[];
	public Number(int iSize)
	{
		 Arr=new int[iSize];

    }
    public void Accept()
    {
    	int i=0;
    	Scanner sobj=new Scanner(System.in);
    	System.out.println("Enter the number of elments:");

    	for(i=0;i<Arr.length;i++)
    	{
    		Arr[i]=sobj.nextInt();
    	}
    }
    public void Display()
    {
        int i = 0;
        System.out.println("Elements of the array are : ");

        for(i = 0 ; i < Arr.length; i++)
        {
            System.out.println(Arr[i]);
        }

    }
    public int Product()
    {
    	int i=0;
    	int iOdd=1;

    	for(i=0;i<Arr.length;i++)
    	{
    		if(Arr[i]%2!=0)
    		{
    		   iOdd=iOdd*Arr[i];
    		}
    	}
    	  return iOdd;
    }
   
  } 
 class ass195
{
	public static void main(String Arg[])
	{

        int iLength=0;
        int iRet=0;
        
		Scanner sobj=new Scanner(System.in);


		System.out.println("Enter the number:");
		iLength=sobj.nextInt();

		Number nobj=new Number(iLength);
		nobj.Accept();
		nobj.Display();
		iRet=nobj.Product();

		System.out.println("Product is:"+iRet);


	}
		

}
